trogdor/src/main/java/org/apache/kafka/trogdor/workload/ConstantThroughputGenerator.java

```diff
@@ -26,17 +26,11 @@
  * The lower the window size, the smoother the traffic will be. Using a 100ms window offers no noticeable spikes in
  * traffic while still being long enough to avoid too much overhead.
  *
- * WARNING: Due to binary nature of throughput in terms of messages sent in a window, each window will send at least 1
- * message, and each window sends the same number of messages, rounded down. For example, 99 messages per second with a
- * 100ms window will only send 90 messages per second, or 9 messages per window. Another example, in order to send only
- * 5 messages per second, a window size of 200ms is required. In cases like these, both the `messagesPerSecond` and
- * `windowSizeMs` parameters should be adjusted together to achieve more accurate throughput.
- *
  * Here is an example spec:
  *
  * {
  *    "type": "constant",
- *    "messagesPerSecond": 500,
+ *    "messagesPerWindow": 50,
```

Comment:
Can you add a comment about what happens if messages per window is 0 (the default)?  (The number of messages is not limited in that case, from what I understand.)

Reply:
Sure, that's reasonable.